NOTICE.md

```diff
@@ -86,15 +86,15 @@ Google Guice / Inject Core API (5.0.1)
 
 * License: Apache License, 2.0
 
-Gradle Wrapper (7.4)
+Gradle Wrapper (8.x)
```

Comment:
Is this still being used?

Reply:
I think we store it in our resources of the wizard bundle and create it when we generate a project based on Gradle, is that right @cdietrich ?

Reply:
yes, this is why this is there in the first place